### DISCUSSION TITLE
Difference between kbar & command palette in Tailwind UI?

### DISCUSSION THREAD

**Discussion by deadcoder0904** [timc1/kbar#178] [General]
I just watched [this video](https://www.youtube.com/watch?v=-jix4KyxLuQ) & I'm wondering if there's any difference between using kbar & Tailwind UI's command palette shown in the video?

Asking bcz I already have installed heroicons, tailwind & headless-ui & don't want to install another dependency if I don't have to.

I couldn't find any differences but you probably know it better :)

PS: It doesn't require a tailwind ui license as it uses all open-source code

**Comment by mhaidarhanif** [comment]
@deadcoder0904 Clearly kbar has it "fully extensible" with a more exposed API rather than building it yourself with Tailwind UI's command palette. 😄

- Simple and straightforward? Go with the video ["Building a command palette with Tailwind CSS, React, and Headless UI" from Tailwind Labs](https://www.youtube.com/watch?v=-jix4KyxLuQ) by @simonswiss
- Need more control and has complex requirements? Go with [kbar](https://kbar.vercel.app) by @timc1  using Tailwind CSS. Especially if you need shortcuts, undo/redo, priority, history, etc.

**Reply by deadcoder0904** [reply]
makes sense! the tailwind command palette does the job pretty well for me but right now i like raycast command paletter more with all the <kbd>tab</kbd> & <kbd>cmd+k</kbd> stuff :)

**Reply by mhaidarhanif** [reply]
Yes, even more, if you want to have a Raycast-like experience, then kbar could be the better starting point.
